src/main/java/pl/programming4you/lectures5/lecture9/Task19/Laptop.java

```diff
@@ -0,0 +1,40 @@
+package pl.programming4you.lectures5.lecture9.Task19;
+
+
+import java.util.Objects;
+
+public class Laptop extends Computer {
+    private final double batteryCapacityWh;
+
+    public Laptop(int RAMGb, int cores, int baseClockMhz, GraphicsCardType GPU, double batteryCapacityWh) {
+        super(RAMGb, cores, baseClockMhz, GPU);
+        this.batteryCapacityWh = batteryCapacityWh;
+    }
+
+    public double getBatteryCapacityWh() {
+        return batteryCapacityWh;
+    }
+
+    // return super
+    @Override
+    public String toString() {
+        return "Laptop: RAM=" + super.getRAMGb() + "GB, CPU=" + super.getCores() + " cores, Clock Speed=" +
+                super.getBaseClockMhz() + "MHz, Graphics=" + super.getGPU() + ", Battery capacity: " +
+                batteryCapacityWh;
```

Comment:
```suggestion
        return super.toString() + ", Battery capacity: " + batteryCapacityWh;
```

In general, you shouldn't use custom toString() methods that much. These autogenerated ones are fine.